recipes/zmqpp/all/conanfile.py

```diff
@@ -0,0 +1,93 @@
+import os
+from conans import ConanFile, CMake, tools
+from conans import AutoToolsBuildEnvironment
+from conans.errors import ConanInvalidConfiguration
+
+class ZmqppConan(ConanFile):
+    name = "zmqpp"
+    homepage = "https://github.com/zeromq/zmqpp"
+    version = "4.2.0"
+    version_major = 4
+    license = "MPLv2"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "This C++ binding for 0mq/zmq is a 'high-level' library that hides most of the c-style interface core 0mq provides."
+    topics = ("conan", "zmq", "0mq", "ZeroMQ", "message-queue", "asynchronous")
+    settings = "os", "compiler", "build_type", "arch"
+    requires = "zeromq/4.3.3"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "make"
+
+    @property
+    def _source_subfolder(self):
+        return os.path.join(self.source_folder, "source_subfolder")
```

Comment:
Typically we do, the root changes from source, build, package

```suggestion
        return "source_subfolder"
```